ext/moderation.py

```diff
@@ -0,0 +1,185 @@
+import disnake
+from disnake.ext import commands
+
+from utils.bot import Cog
+from utils.checks import convert
+
+
+class moderation(Cog):
+    @commands.slash_command(name="ban")
+    @commands.has_permissions(ban_members=True)
+    async def ban(
+        self, inter: disnake.ApplicationCommandInteraction, member: disnake.Member, reason: str = "No reason"
+    ):
+        """Bans a member
+
+        Parameters
+        -----------
+        member: The member to ban
+        reason: The reason for banning
+        """
+        await inter.send(
+            embed=disnake.Embed(
+                title="Member banned",
+                description=f"⚒ | {member.mention} has been wiped out from this server by {inter.user.mention} for **{reason}**!",
+                color=disnake.Color.red(),
+            )
+        )
+        await member.send(
+            embed=disnake.Embed(
+                title="Banned",
+                description=f"You have been banned from {inter.guild.name} by {inter.user.mention} for **{reason}**!",
+            )
+        )
+        await inter.guild.ban(member)
```

Comment:
1. There should be a condition that checks if member's top role is lower than command invoker's role
2. No handler for ban failure and DM message
3. Instead of "wiped out" just use banned

^ this applies to almost every command